packages/headless/src/index.ts

```diff
@@ -84,3 +84,4 @@ export {buildSearchParameterSerializer} from './features/search-parameters/searc
 export * as HighlightUtils from './utils/highlight';
 export {HighlightKeyword} from './utils/highlight';
 export {VERSION} from './utils/version';
+export {BaseFacetValue} from './features/facets/facet-api/response';
```

Comment:
It's used a lot inside the Headless Breadcrumb Manager without being exported. It is ok to export? If yes I should document it as well...

Reply:
I'd prefer to not expose such an implementation detail, keeping it possible to adjust the properties on the interface or rename the interface in a non-breaking way.

Looking at how it's used, I'd prefer to either keep importing it from dist, or explore adjusting the approach, using the higher-level types `FacetBreadcrumb`, `NumericFacetBreadcrumb` etc. types as building blocks.

Reply:
Okay, I will switch back to the dist type 👍